enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,337 @@
+---
+title: alerts-ui-managment
+authors:
+  - "@sradco"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2025-07-24
+last-updated: 2025-07-30
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-46597"
+  - "https://issues.redhat.com/browse/CNV-62972"
+---
+# New APIs to support Alerts UI Management in OpenShift
+
+## Summary
+Provide a user‑friendly UI and REST API for defining, viewing, editing, disabling/enabling and silencing Prometheus alerts without manual YAML edits, reducing alert fatigue and improving operational efficiency.
+Platform alerts will be updated via entries in the `AlertRelabelConfig` CR (in `cluster-monitoring-config`) rather than editing their original `PrometheusRule`.
+
+## Motivation
+- The current `PrometheusRule` CRD + `oc apply` workflow is powerful yet cumbersome for day‑to‑day alert tweaks.
```

Comment:
We should mention AlertingRule + AlertRelabelConfig CRDs rather than PrometheusRule IMHO. The UI mismatch should also be mentioned.
```suggestion
- While it's possible to customize built-in alerting rules with the `AlertingRule` + `AlertRelabelConfig` CRDs, the process is cumbersome and error-prone. It requires creating YAML manifests manually and there's no practical way to verify the correctness of the configuration.
- The OCP console doesn't take into account alerting customization: it will display built-in alerting rules rather than hiding them or marking them as overriden. 
```